SITnett/graphics/style.css

```diff
@@ -1,8 +1,67 @@
+
 body {
 	color: white;
 	background-color: gray;
 }
 
 a {
 	color: black;
+}
+
+/* Mobile phones first */
+
+.topnav {
+	overflow: hidden;
+	background-color: gray;
+	position: relative;
+}
+
+/* Hide the links inside the navigation menu (except for logo/home) */
+.topnav #menuLinks {
+	display: none;
+}
+
+/* Style navigation menu links */
+.topnav a {
+	color: white;
+	padding: 14px 16px;
+	text-decoration: none;
+	font-size: 17px;
+	display: block;
+}
+
+/* Style the hamburger menu */
+.topnav a.icon {
+	background: gray;
+	display: block;
+	position: absolute;
+	right: 0;
+	top: 0;
+}
+
+/* Add a grey background color on mouse-over */
+.topnav a:hover {
+	color: #ed1c24;
```

Comment:
Vi kan la denne stå inntil videre, men jeg tror vi gjør oss selv en stor tjeneste om vi oppretter CSS-variabler for farger og legger dem i `:root` øverst i CSS-en. Kan legge det inn i Trello.

Reply:
Oki, da lar jeg det stå for nå!